VAMobile/src/api/demographics/getDemographics.tsx

```diff
@@ -0,0 +1,29 @@
+import { useQuery } from '@tanstack/react-query'
+
+import { DemographicsPayload, UserDemographics } from '../types/DemographicsData'
```

Comment:
Good eyes, VS Code auto generated the imports for me, so I didn't pay much attention to them. Not sure if we have a CU ticket for this, but it would be useful to have linting catch relative imports. When I was on VSP, we had a custom ESLint rule for that in the vets-website repo

Reply:
Searched import and relative and found nothing so [here's](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/6678) the new ticket.